entries/server-entry.js

```diff
@@ -64,17 +67,21 @@ export async function start({port, dir = '.'} /*: any */) {
   });
 
   return new Promise(resolve => {
-    server.listen(port, () => {
-      // eslint-disable-next-line no-console
-      resolve(server);
-    });
+    server &&
+      server.listen(port, () => {
+        // eslint-disable-next-line no-console
```

Comment:
Just noticed that this eslint thing can be removed (not related to your change though).